### ISSUE TITLE
CGI script not compatible with 1.0.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#203]
```
I guess jsupload.cgi.pl script does not reflect the changes introduced in the 
commit 
https://github.com/manolo/gwtupload/commit/a22d3b89805269684ab89013bb73f695d86fa
ce7#diff-22a829bcbe5759e987b7f138f72c379cR2 of file UConsts.java.

I have applied the attached patch and got my application working in Chrome but 
I still have problems with IE.

```

Original issue reported on code.google.com by `bahri.ge...@gmail.com` on 29 Oct 2014 at 2:52

Attachments:
- [jsupload_xml_patch.diff](https://storage.googleapis.com/google-code-attachments/gwtupload/issue-203/comment-0/jsupload_xml_patch.diff)

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 16d283771a77.
```

Original comment by `manuel.carrasco.m` on 7 Jan 2015 at 3:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Applied your patch.

Related with IE, can you elaborate the issue, preferably in a new ticket.

Thanks for contributing.
```

Original comment by `manuel.carrasco.m` on 7 Jan 2015 at 3:36
- Added labels: **Milestone-1.0.2**
